### PR TITLE
Add user roles for image/version/org update and delete

### PR DESCRIPTION
**Pull request by Minoli**

### CODE REVIEW

File: components/portal/src/components/image/index.js [hunk 1]
```diff
@@ -353,7 +353,7 @@ class Image extends React.Component {
                             </Typography>
                         </div>
                         {
-                            globalState.get(StateHolder.USER).roles.includes(Constants.Permission.PUSH)
+                            (imageData.userRole === Constants.Permission.PUSH)
```
Comment (nadundesilva) [inline components/portal/src/components/image/index.js]: Need to check whether the userRole is push or admin both

File: components/portal/src/components/image/version/VersionDeleteDialog.js [hunk 2]
```diff
@@ -44,11 +44,11 @@ class VersionDeleteDialog extends React.Component {
...
     handleDeleteVersion = () => {
         const self = this;
-        const {globalState, image, version} = self.props;
-        NotificationUtils.showLoadingOverlay(`Deleting image ${image}`, globalState);
+        const {globalState, org, image, version} = self.props;
+        NotificationUtils.showLoadingOverlay(`Deleting image ${org}/${image}`, globalState);
         HttpUtils.callHubAPI(
             {
-                url: `/images/${image}/${version}`,
+                url: `/images/${org}/${image}/${version}`,
```
Comment (nadundesilva) [inline components/portal/src/components/image/version/VersionDeleteDialog.js]: Shouldn't this URL point to /artifacts/${org}/${image}/${version}

Comment (Minoli) [reply components/portal/src/components/image/version/VersionDeleteDialog.js]: fixed

File: components/portal/src/components/image/index.js (hunk 1 above)
Comment (Minoli) [reply components/portal/src/components/image/index.js]: fixed

**Pull request merged**